### PR TITLE
Getty shortcode: fix all PHPCS warnings in preparation for Fusion

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Fix all PHPCS warnings in preparation for Fusion

#### Testing instructions:

* Add the following content to a new post:
`[getty src="82278805"]`
`<div class="getty embed image" style="background-color:#fff;display:inline-block;font-family:'Helvetica Neue',Helvetica,Arial,sans-serif;color:#a7a7a7;font-size:11px;width:100%;max-width:462px;"><div style="padding:0;margin:0;text-align:left;"><a href="http://www.gettyimages.com/detail/82278805" target="_blank" style="color:#a7a7a7;text-decoration:none;font-weight:normal !important;border:none;display:inline-block;">Embed from Getty Images</a></div><div style="overflow:hidden;position:relative;height:0;padding:80.086580% 0 0 0;width:100%;"><iframe src="//embed.gettyimages.com/embed/82278805?et=jGiu6FXXSpJDGf1SnwLV2g&sig=TFVNFtqghwNw5iJQ1MFWnI8f4Y40_sfogfZLhai6SfA=" width="462" height="370" scrolling="no" frameborder="0" style="display:inline-block;position:absolute;top:0;left:0;width:100%;height:100%;"></iframe></div><p style="margin:0;"></p></div>`
* Make sure the embeds work well.

#### Proposed changelog entry for your changes:

* None

### CODE REVIEW

File: modules/shortcodes/getty.php [hunk 1]
```diff
@@ -146,8 +152,10 @@ function wpcom_shortcodereverse_getty( $content ) {
 			if ( ! empty( $height ) ) {
 				$shortcode .= ' height="' . esc_attr( $height ) . '"';
 			}
-			// While it does not appear to have any practical impact, Getty has
-			// requested that we include TLD in the embed request
+			/**
```
Comment (mdawaffe) [inline modules/shortcodes/getty.php]: I don't think this should be a doc block. Should it?

```suggestion
			/*
```

**Review by mdawaffe — COMMENTED** [review]
Love it.

**Comment by mdawaffe** [comment]
I tested this PR and this PR plus the diff from [above](https://github.com/Automattic/jetpack/pull/11629#discussion_r267499021). Both work in the Classic Visual Editor and the Classic Text Editor.

Pasting the `<div...` content into the block editor does not work, but this PR does not intend to change that behavior.

File: modules/shortcodes/getty.php [hunk 2]
```diff
@@ -87,12 +94,11 @@ function wpcom_shortcodereverse_getty( $content ) {
 	}
 
 	$regexp     = '!<iframe\s+src=[\'"](https?:)?//embed\.gettyimages\.com/embed(/|/?\?assets=)([a-z0-9_-]+(,[a-z0-9_-]+)*)[^\'"]*?[\'"]((?:\s+\w+=[\'"][^\'"]*[\'"])*)((?:[\s\w]*))></iframe>!i';
-	$regexp_ent = str_replace( '&amp;#0*58;', '&amp;#0*58;|&#0*58;', htmlspecialchars( $regexp, ENT_NOQUOTES ) );
+	$regexp_ent = str_replace( '&amp;#0*58;', '&amp;#0*58;|&#0*58;', htmlspecialchars( $regexp, ENT_NOQUOTES ) ); // phpcs:ignore
```
[the remark this answers is not in the document]
Comment (jeherve) [reply modules/shortcodes/getty.php]: Done in 31caf10465efd7880e426719a528fe08175b66f2

**Review by mdawaffe — APPROVED** [review]

**Pull request merged**